db/schema.rb

```diff
@@ -24,6 +24,25 @@
     t.datetime "updated_at", null: false
   end
 
+  create_table "invoices", force: :cascade do |t|
+    t.json "discount"
+    t.json "adjustment"
```

Comment:
Please see if we will need to perform queries or back end validation on any of the json fields. Because it is not very practical to use a json field for data which will need frequent changes or operations.